src/Microsoft.Identity.Web/MicrosoftIdentityCircuitHandler.cs

```diff
@@ -17,257 +17,257 @@
+	/// </summary>
+	public static class MicrosoftIdentityBlazorServiceCollectionExtensions
+	{
+		/// <summary>
+		/// Add the incremental consent and conditional access handler for Blazor
+		/// server side pages.
+		/// </summary>
+		/// <param name="builder">Service side blazor builder.</param>
+		/// <returns>The builder.</returns>
+		public static IServerSideBlazorBuilder AddMicrosoftIdentityConsentHandler(
+			this IServerSideBlazorBuilder builder)
+		{
+			if (builder == null)
+			{
+				throw new ArgumentNullException(nameof(builder));
+			}
 
-            builder.Services.TryAddEnumerable(ServiceDescriptor.Scoped<CircuitHandler, MicrosoftIdentityServiceHandler>());
-            builder.Services.TryAddScoped<MicrosoftIdentityConsentAndConditionalAccessHandler>();
-            return builder;
-        }
+			builder.Services.TryAddEnumerable(ServiceDescriptor.Scoped<CircuitHandler, MicrosoftIdentityServiceHandler>());
+			builder.Services.TryAddScoped<MicrosoftIdentityConsentAndConditionalAccessHandler>();
+			return builder;
+		}
 
-        /// <summary>
-        /// Add the incremental consent and conditional access handler for
-        /// web app pages, Razor pages, controllers, views, etc...
-        /// </summary>
-        /// <param name="services">Service collection.</param>
-        /// <returns>The service collection.</returns>
-        public static IServiceCollection AddMicrosoftIdentityConsentHandler(
-            this IServiceCollection services)
-        {
-            if (services == null)
-            {
-                throw new ArgumentNullException(nameof(services));
-            }
+		/// <summary>
+		/// Add the incremental consent and conditional access handler for
+		/// web app pages, Razor pages, controllers, views, etc...
+		/// </summary>
+		/// <param name="services">Service collection.</param>
+		/// <returns>The service collection.</returns>
+		public static IServiceCollection AddMicrosoftIdentityConsentHandler(
+			this IServiceCollection services)
+		{
+			if (services == null)
+			{
+				throw new ArgumentNullException(nameof(services));
+			}
 
-            services.TryAddEnumerable(ServiceDescriptor.Scoped<CircuitHandler, MicrosoftIdentityServiceHandler>());
-            services.TryAddScoped<MicrosoftIdentityConsentAndConditionalAccessHandler>();
-            return services;
-        }
-    }
+			services.TryAddEnumerable(ServiceDescriptor.Scoped<CircuitHandler, MicrosoftIdentityServiceHandler>());
+			services.TryAddScoped<MicrosoftIdentityConsentAndConditionalAccessHandler>();
+			return services;
+		}
+	}
 
-    /// <summary>
-    /// Handler for Blazor specific APIs to handle incremental consent
-    /// and conditional access.
-    /// </summary>
+	/// <summary>
+	/// Handler for Blazor specific APIs to handle incremental consent
+	/// and conditional access.
+	/// </summary>
 #pragma warning disable SA1402 // File may only contain a single type
```

Comment:
Can the type be moved to its own file in order to make it easier to find on the github folder view?